content/components/button.mdx

```diff
@@ -436,6 +436,12 @@ There are rare cases where it's ok to disable a button, but it should generally
 
 For more information on disabled controls, see GitHub's [link and button a11y guidance](https://accessibility-playbook.github.com/link-and-button-guidance#disabling-links-and-buttons) (GitHub staff only).
 
+### Inactive buttons and `aria-disabled`
+
+An [inactive](#inactive) button **should not be conveyed as disabled** with `aria-disabled` if it performs an action when activated. For example, showing a dialog with more info about why the button is inactive.
+
+An [inactive](#inactive) button **may be conveyed as disabled** with `aria-disabled` if it does not perform an action when activated.
```

Comment:
Would the component determine if `aria-disabled` should be applied in cases where an inactive state is used without an action, or is that the responsibility of the developer? If it's the latter, _should_ we make this part of the component?

Reply:
It's the latter. We could make it part of the component.